lib/sequelize.js

```diff
@@ -143,7 +144,8 @@ var Sequelize = function(database, username, password, options) {
     quoteIdentifiers: true,
     hooks: {},
     isolationLevel: Transaction.ISOLATION_LEVELS.REPEATABLE_READ,
-    databaseVersion: 0
+    databaseVersion: 0,
+    validation: false
```

Comment:
perhaps `typeValidation` and perhaps allow it on model level aswell?
`validation` is perhaps slightly generic.